pkg/controllers/scheduler/schedulingunit.go

```diff
@@ -152,6 +154,24 @@ func schedulingUnitForFedObject(
 		schedulingUnit.MaxClusters = maxClustersOverride
 	}
 
+	gvk := schedulingUnit.GroupVersion.WithKind(schedulingUnit.Kind)
+	_, ok := fedpodutil.PodSpecPaths[gvk.GroupKind()]
+	if !ok {
+		return schedulingUnit, nil
+	}
+	podSpec, err := fedpodutil.GetResourcePodSpec(fedObject, gvk)
+	if err != nil {
+		return nil, err
+	}
+
+	resourceRequest := *framework.NewResource(resourceutil.GetPodResourceRequests(podSpec))
```

Comment:
You can use `getResourceRequest(typeConfig, fedObject)` here

Reply:
And shall we move `getResourceRequest` to util.go?

Reply:
I'd like to move it to util.go because it can be used both in schedulingtriggers and scheduler.